### DISCUSSION TITLE
Are the patient_ids of different contrast for the same patient in HDF5 files of Brain?

### DISCUSSION THREAD

**Discussion by bhl1** [facebookresearch/fastMRI#314] [Q&A]
I  find the FastMRI paper mentions that every patient has a unique patient_id.
![image](https://github.com/facebookresearch/fastMRI/assets/38810307/91a0d8d1-1ba3-49a9-b2cf-9e3a2fd60251)

![image](https://github.com/facebookresearch/fastMRI/assets/38810307/eca7fdaa-edc7-4516-8f9b-e076ad15ca35)

So can we find the paired data of the same patient with different contrasts through the patient_id

**Comment by mmuckley** [comment]
Hello @bhl1, in principle yes, but practically I don't think you can do it with fastMRI. To my knowledge NYU fully sampled **one contrast for each patient**. This was done to keep the costs under control for lengthened protocols across the hospital.
